azure-pipelines.yml

```diff
@@ -787,26 +787,29 @@ jobs:
       name: 'linux-pool'
       demands: assignment -equals default
     variables:
-      pr.num: $[ variables['System.PullRequest.PullRequestNumber'] ]
       branch_sha: $[ dependencies.git_sha.outputs['out.branch'] ]
       status: $[ dependencies.collect_build_data.result ]
       is_release: $[ dependencies.check_for_release.outputs['out.is_release'] ]
     steps:
+      - checkout: self
+        persistCredentials: true
+      - template: ci/bash-lib.yml
+        parameters:
+          var_name: bash-lib
       - bash: |
           set -euo pipefail
 
-          tell_daml() {
-              local message pr_handler
-              message=$1
-              pr_handler=$(head -1 release/rotation | awk '{print $1}')
-              curl -XPOST \
-                   -i \
-                   -H 'Content-Type: application/json' \
-                   --data "{\"text\":\"<@${pr_handler}> <https://dev.azure.com/digitalasset/daml/_build/results?buildId=$(Build.BuildId)|Build $(Build.BuildId)> for release PR <https://github.com/digital-asset/daml/pull/$(pr.num)|#$(pr.num)> has completed with status ${message}.\"}" \
-                   $(Slack.team-daml)
-          }
+          source $(bash-lib)
+
+          AUTH="$(get-gh-auth-header)"
+          PR=$(curl -H "$AUTH" \
+                    -H "Accept: application/vnd.github.groot-preview+json" \
+                    -s \
+                    "https://api.github.com/repos/digital-asset/daml/commits/$(git rev-parse HEAD)/pulls" \
+               | jq '.[0].number')
```

Comment:
What happens if someone pushes to the branch between this triggering and running?

Reply:
I suppose GitHub would return an empty list and `jq` will explode. Worst case we don't get a notification on Slack, though, and I really don't expect anyone to deliberately sabotage this like that.

Reply:
OK, if that's not expected behavior, I'm happy.